kwc-challenge-set.html

```diff
@@ -371,6 +420,18 @@ <h3 class="title-title">
                 buttonLabel: {
                     type: String,
                     computed: '_computeButtonLabel(_completed, progress)'
+                },
+                buttonLabel: {
+                    type: String,
+                    computed: '_computeButtonLabel(_completed, progress)'
+                },
+                /**
+                 * Flags if challenge-friend button is hidden
+                 * @type {Boolean}
+                 */
+                _hideChallengeFriend: {
```

Comment:
I guess if this is something we'll be setting from "outside" this should not be prefixed with a `_`, no?

Reply:
Totes :) Thanks!